profiler/profiler_test.go

```diff
@@ -521,3 +521,43 @@ func TestImmediateProfile(t *testing.T) {
 	case <-received:
 	}
 }
+
+func TestExecutionTrace(t *testing.T) {
+	got := make(chan profileMeta)
+	server := httptest.NewServer(&mockBackend{t: t, profiles: got})
+	defer server.Close()
+
+	t.Setenv("DD_PROFILING_EXECUTION_TRACE_ENABLED", "true")
+	t.Setenv("DD_PROFILING_EXECUTION_TRACE_FREQUENCY", "3s")
+	t.Setenv("DD_PROFILING_EXECUTION_TRACE_DURATION", "50ms")
```

Comment:
NIT: Probably a good idea to reset these env vars with a defer to their previous value. I know we're not doing this everywhere right now, but arguably any test modifying global state should restore it back to how it found it.

Reply:
[`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) already resets the vars at the end of the test.

Reply:
TIL!